src/app/frontend/podexec/afterrender_directive.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Returns directive that runs code after a template render.
+ *
+ * @return {!angular.Directive}
+ * @ngInject
+ */
+export default function afterRenderDirective($timeout) {
+  var def = {
+    restrict: 'A',
+    terminal: true,
+    transclude: false,
+    /** @param {!AfterRender.attrs} attrs */
+    link: function (scope, element, attrs) {
```

Comment:
This probably doesn't have to be generic. Would appreciate some pointers as to how to clean this up a bit.